app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_years, dog_years, expected_result",
+    [
+        pytest.param(0, 0, [0, 0], id="test with arguments equal 0"),
+        pytest.param(14, 14, [0, 0], id="test with arguments 14"),
+        pytest.param(16, 14, [1, 0], id="test with different arguments"),
+        pytest.param(23, 23, [1, 1], id="test with arguments equal 23"),
+        pytest.param(24, 24, [2, 2], id="test with arguments equal 24"),
+        pytest.param(28, 28, [3, 2], id="test with arguments equal 28"),
+        pytest.param(100, 100, [21, 17], id="test with arguments equal 100"),
```

Comment:
what about negative and big numbers? Last `param` can be considered as a test for really big numbers. But change `id`
 What about test case for incorrect data type passing to your function ?